pytorch/inference/docker/1.12/py3/cu116/Dockerfile.gpu

```diff
@@ -2,10 +2,10 @@ ARG PYTHON=python3
 ARG PYTHON_VERSION=3.8.13
 
 # PyTorch Binaries
-ARG PT_INFERENCE_URL=https://aws-pytorch-unified-cicd-binaries.s3.us-west-2.amazonaws.com/r1.12.0_e3/20220628-223427/c1e155d3cbd8d4115c80866bed4c717775363e7a/torch-1.12.0%2Bcu116-cp38-cp38-linux_x86_64.whl
-ARG PT_TORCHVISION_URL=https://download.pytorch.org/whl/cu116/torchvision-0.13.0%2Bcu116-cp38-cp38-linux_x86_64.whl
-ARG PT_TORCHAUDIO_URL=https://download.pytorch.org/whl/cu116/torchaudio-0.12.0%2Bcu116-cp38-cp38-linux_x86_64.whl
-ARG PT_TORCHDATA_URL=https://download.pytorch.org/whl/test/torchdata-0.4.0-cp38-cp38-linux_x86_64.whl
+ARG PT_INFERENCE_URL=https://aws-pytorch-unified-cicd-binaries.s3.us-west-2.amazonaws.com/r1.12.1_e3/20220826-044615/579ffd0893a9d3e60773918bef0b000dfe60a7ce/torch-1.12.1%2Bcu116-cp38-cp38-linux_x86_64.whl
```

Comment:
We are no longer using the "e3" name and are now using "ec2" instead. Can you please change the name of these artifacts to reflect that change?